.github/workflows/ci-elasticsearch-otel.yml

```diff
@@ -0,0 +1,53 @@
+name: CIT OTEL Elasticsearch
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  elasticsearch-otel-5:
```

Comment:
could we use matrix for this instead of repeating job specs?

Reply:
Ah right. Sorry I completely forgot about it.